README.md

```diff
@@ -57,31 +58,30 @@ app.listen(3000);
 
 You're defining an interceptor that will be executed whenever the response contains html as Content-Type. If this is true, it will append a child at the end of the body.
 
-Find other examples at [/examples folder](https://github.com/axiomzen/express-interceptor/tree/master/examples).
+See [more examples](https://github.com/axiomzen/express-interceptor/tree/master/examples).
 
 ## API
 
-* * `isInterceptable()`: is a predicate function where you define a condition whether or not to intercept a response. Returning `true` buffers the request, and proceeds calling `intercept()` as well as `afterSend()`. Typically, you want to check for this condition in the `res` object in the definition of the middleware.
+* `isInterceptable()` (required): is a predicate function where you define a condition whether or not to intercept a response. Returning `true` buffers the request, and proceeds calling `intercept()` as well as `afterSend()`. Typically, you want to check for this condition in the `res` object in the definition of the middleware.
 
-* * `intercept(body, send)`: Enables you to process the complete response in `body`, as a  properly encoded String. When you're finished with what you wish to do, call `send(newBody)` passing `newBody` as the content you wish to send back to the client.
+* `intercept(body, send)` (required): Enables you to process the complete response in `body`, as a  properly encoded String. When you're finished with what you wish to do, call `send(newBody)` passing `newBody` as the content you wish to send back to the client.
 
 * `afterSend(oldBody, newBody)`: This method will be called after sending the response to the client – after the `done()` callback in the `send()` method is executed. This method would typically be used to cache something, log stats, fire a job, etc.
 
-(*) Required methods.
 
 ## Similar to
 
 - [express-hijackresponse](https://github.com/papandreou/express-hijackresponse)
-Has issues with cache; code is difficult to maintain. 
```

Comment:
Instead of saying "code is difficult to maintain" say "code is no longer maintained and it relies too much on node internals for us to want to fork it and maintain it"